### ISSUE TITLE
Option to change the refresh rate of the animations

### ISSUE THREAD

**Issue by 9cento** [Open-Shell/Open-Shell-Menu#1553]
### Is your feature request related to a problem? Please describe.

I have a 165hz monitor and as a result the default fps of Open-Shell's animations fells choppy. I already searched left and right in the settings but there's no dedicated entry, slowing down animations' time helps a bit but it's not an ideal solution to the root issue

### Describe the solution you'd like

It would be ideal instead to have a dedicated setting for that, and a temporary workaround in the meantime involving editing the registry or some textual config file would be more than welcome

### Area of issue

Start menu

**Comment by ge0rdi** [comment]
I assume you are referring to _main_ / _sub_ menu animation.
What type of animation do you use (fade/slide/random)?

Can you describe "animations fells choppy" a bit more?
Maybe provide some captured vide of the animation that would show the issue?

**Comment by ge0rdi** [comment]
@9cento 
Could you eventually try build from PR #1555 ?
It contains improved animation timing. Maybe it will help in your case.

**Comment by 9cento** [comment]
@ge0rdi yes I'm referring to main and sub menu animations, I use the slide animation personally but I think it's more like a global issue.
I can't screencap a video atm but anyway it wouldn't be of much help if viewed on a 60 fps screen because the problem is exactly that: the animations are consistent, but their fps is low (at least for a 165hz monitor), there's no better way to explain it.
Having an option to set their fps would be cool, I'll try PR #1555 asap anyway and report back.
Thank you for now.

**Comment by 9cento** [comment]
@ge0rdi just tried PR #1555
It feels a little better but it could be placebo, I still think there's room for improvement with an fps chooser.
Thank you for your effort as always.

**Comment by ge0rdi** [comment]
> the animations are consistent, but their fps is low

`Open-Shell` redraws start window (with either changed alpha or position) in a loop until configured time elapsed. Thus it depends on how fast can GDI redraw content of the window. It may depend on machine power, skin complexity, etc.

On my machine the _fade_ option does like 400fps and the _slide_ option like 220fps. Actually that's way too much (I have 144Hz display) so it would be better if `Open-Shell` throttles it to not spend unnecessary resources.

There is new build in PR #1555 that logs actual fps of menu animation (based on how many redraws were done during given time period). You can try it and enable `Open-Shell` logging using this [guide](https://github.com/Open-Shell/Open-Shell-Menu/wiki/How-to-obtain-advanced-Open-Shell-Menu-log).

Log will then contain line like:
`Menu animation 44 frames in 200ms (220 fps)`

**Comment by 9cento** [comment]
>You can try it and enable Open-Shell logging using this [guide](https://github.com/Open-Shell/Open-Shell-Menu/wiki/How-to-obtain-advanced-Open-Shell-Menu-log).
Log will then contain line like:
Menu animation 44 frames in 200ms (220 fps)

I will, but it's really strange since I have a 4090 and the default windows start menu still opens smoother
